variants/classicalcrowded/classicalcrowded.go

```diff
@@ -0,0 +1,534 @@
+package classicalcrowded
+
+import (
+	"github.com/zond/godip"
+	"github.com/zond/godip/graph"
+	"github.com/zond/godip/state"
+	"github.com/zond/godip/phase"
+	"github.com/zond/godip/variants/classical"
+	"github.com/zond/godip/variants/common"
+)
+
+const (
+	Balkans       godip.Nation = "Balkans"
+	Benelux       godip.Nation = "Benelux"
+	Iberia        godip.Nation = "Iberia"
+	Scandinavia   godip.Nation = "Scandinavia"
+)
+
+var (
+	Nations    = []godip.Nation{godip.Austria, Balkans, Benelux, godip.England, godip.France, godip.Germany, Iberia, godip.Italy, godip.Turkey, godip.Russia, Scandinavia}
+	PhaseTypes = []godip.PhaseType{godip.Movement, godip.Retreat, godip.Adjustment}
+	Seasons    = []godip.Season{godip.Spring, godip.Fall}
+	UnitTypes  = []godip.UnitType{godip.Army, godip.Fleet}
+	SVGUnits   = classical.SVGUnits
+	SVGFlags = map[godip.Nation]func() ([]byte, error){
+		Balkans: func() ([]byte, error) {
+			return Asset("svg/balkans.svg")
+		},
+		Benelux: func() ([]byte, error) {
+			return Asset("svg/benelux.svg")
+		}
+	}
+	Parser = classical.Parser
+)
```

Comment:
I think the only special variables you need to declare in this block are `Nations` and `SVGFlags`. All the others you can just reference as `classical.Whatever`. I think that you need to do something slightly different for `SVGFlags`, otherwise you won't get the seven standard flags included.